.github/discord-embed-webhook.py

```diff
@@ -0,0 +1,66 @@
+import argparse
+import os
+import json
+import pathlib
+
+from requests import Response
+from discord_webhook import DiscordEmbed, DiscordWebhook
+
+
+GITHUB_CONTEXT_JSON = os.environ.get("GITHUB_CONTEXT_JSON", "{}")
+DOWNLOAD_BASE_URL_TEMPLATE = (
+    "https://build.livepeer.live/go-livepeer/{version}/{filename}"
+)
+
+
+def get_embeds(embed: DiscordEmbed, ref_name: str, checksums: list[str]):
+    for line in checksums:
+        _, filename = line.split()
+        download_url = DOWNLOAD_BASE_URL_TEMPLATE.format(
+            version=ref_name,
+            filename=filename,
+        )
+        # strip the initial `livepeer-` prefix and then identify build
+        # platform/architecture/release-tag values from file name
+        title = filename[9:].split(".")[0]
```

Comment:
I see. I think the comment and code could be replaced with [lstrip()](https://docs.python.org/3/library/stdtypes.html#str.lstrip)
```python
title = filename.lstrip('livepeer-').split(".")
```

Reply:
done*